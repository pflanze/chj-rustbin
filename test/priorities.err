WARNING: error parsing the file name of path "test/priorities/2025-01-20_015025_Mon-another OPEN{1-24} OPEN{today}": adjacent opening markers found without a closing marker inbetween: ParseableStr { position: 30, backing: "2025-01-20_015025_Mon-another OPEN{1-24} OPEN{today}", s: "OPEN" } then ParseableStr { position: 41, backing: "2025-01-20_015025_Mon-another OPEN{1-24} OPEN{today}", s: "OPEN" } at "OPEN{today}"
WARNING: error parsing the file name of path "test/priorities/another OPEN{2026-1-18, importance: 1}": expected digit as part of month number at "another OPEN{2026-1-18, importance: 1}"
WARNING: duplicate task for key NaiveDateTime(2024-09-30T11:13:32): "test/priorities/erroneous/2024-09-30_111332_Mon-duplicate OPEN" "test/priorities/erroneous/2024-09-30_111332_Mon-duplicate 2 OPEN"
WARNING: duplicate task for key NaiveDateTime(2024-09-30T11:13:32): "test/priorities/erroneous/2024-09-30_111332_Mon-foo OPEN{d: [2024-09-27]}" "test/priorities/erroneous/2024-09-30_111332_Mon-duplicate OPEN"
WARNING: unknown dependency String(Owned("2024-09-27_201849")) in: "test/priorities/2024-09-27_201910_Fri-do bar OPEN{prio: 1, depends: 2024-09-27_201849}"
WARNING: unknown dependency String(Inline("reprocessing")) in: "test/priorities/2024-09-27_212233_Fri-do OPEN{d: reprocessing}"
WARNING: unknown dependency String(Owned("2024-09-27_201931_")) in: "test/priorities/erroneous/2024-09-30_111329_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931_Mon]}"
WARNING: unknown dependency String(Owned("2024-09-27_201931_Mon")) in: "test/priorities/erroneous/2024-09-30_111329_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931_Mon]}"
WARNING: unknown dependency String(Owned("2024-09-27_201931")) in: "test/priorities/erroneous/2024-09-30_111330_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931]}"
WARNING: unknown dependency String(Owned("2024-09-27_201931_")) in: "test/priorities/erroneous/2024-09-30_111330_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931]}"
WARNING: unknown dependency String(Inline("foobar")) in: "test/priorities/erroneous/2024-09-30_111331_Mon-foo OPEN{d: [foobar]}"
WARNING: unknown dependency String(Inline("2024-09-27")) in: "test/priorities/erroneous/2024-09-30_111332_Mon-foo OPEN{d: [2024-09-27]}"
12 error(s) shown as warnings
