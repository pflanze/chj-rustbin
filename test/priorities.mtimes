1727947274	test/priorities/2024-09-27_201849_Fri-do foo OPEN
1727947274	test/priorities/2024-09-27_201910_Fri-do bar OPEN{prio: 1, depends: 2024-09-27_201849}
1727947274	test/priorities/2024-09-27_201931_Fri-notes on issue baz
1727959766	test/priorities/2024-09-27_212233_Fri-do OPEN{d: reprocessing}
1727947274	test/priorities/2024-09-29_231231_Sun-meta another task OPEN{d: [2024-09-27_201849_Fri-do, 2024-09-27_201931_Fri ], prio: 2024-11-01 12:00}
1729456986	test/priorities/2024-10-02_193929_Wed-some OPEN{occasionally}
1729367175	test/priorities/2024-10-02_200923_Wed-some OPEN{ today , d:2024-09-29_231231_Sun-meta}
1732542859	test/priorities/2024-11-25_145417_Mon-some OPEN{1} DONE
1733195978	test/priorities/2024-12-02_203000_Mon-some OPEN{12-3}
1737334224	test/priorities/2025-01-20_015023_Mon-another OPEN{1-24} DONE OPEN{today}
1737334224	test/priorities/2025-01-20_015024_Mon-another OPEN{1-24} DONE
1737334224	test/priorities/2025-01-20_015025_Mon-another OPEN{1-24} OPEN{today}
1737334224	test/priorities/2025-01-20_015026_Mon-another OPEN{1-24} WONTFIX
1737334224	test/priorities/2025-01-20_015027_Mon-another OPEN{1-24, importance: 1}
1737334224	test/priorities/2025-01-20_015028_Mon-another OPEN{2025-1-24, importance: 1}
1737334224	test/priorities/2025-01-20_015029_Mon-another OPEN{01-18, importance: 1}
1737334224	test/priorities/2025-01-20_015030_Mon-another OPEN{2026-1-18, importance: 1}
1730291051	test/priorities/README
1737334224	test/priorities/another OPEN{2026-1-18, importance: 1}
1727947274	test/priorities/erroneous/2024-09-29_231214_Sun-meta another task OPEN{depends: 2024-09-27_201849_Fri, 2024-09-27_201931_Fri}
1727947274	test/priorities/erroneous/2024-09-30_111329_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931_Mon]}
1727947274	test/priorities/erroneous/2024-09-30_111330_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931]}
1727947274	test/priorities/erroneous/2024-09-30_111331_Mon-foo OPEN{d: [foobar]}
1727947274	test/priorities/erroneous/2024-09-30_111332_Mon-duplicate 2 OPEN
1727947274	test/priorities/erroneous/2024-09-30_111332_Mon-duplicate OPEN
1727947274	test/priorities/erroneous/2024-09-30_111332_Mon-foo OPEN{d: [2024-09-27]}
1730291144	test/priorities/foo/2024-10-24_153947_Thu-foo (for Wed OPEN{10-27})
1730291249	test/priorities/foo/done/2024-10-30_132705_Wed-foo with invalid open but ignored OPEN{2024-10-27}
1730291249	test/priorities/foo/obsolete/2024-10-30_132704_Wed-foo with invalid open but ignored OPEN{2024-10-27}
1727969773	test/priorities/reprocessing/2024-10-03_173610_Thu-README OPEN{prio: 2}
