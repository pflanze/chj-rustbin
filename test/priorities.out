-inf	test/priorities/2024-09-27_201849_Fri-do foo OPEN
-inf	test/priorities/erroneous/2024-09-29_231214_Sun-meta another task OPEN{depends: 2024-09-27_201849_Fri, 2024-09-27_201931_Fri}
-3.24	test/priorities/2024-09-29_231231_Sun-meta another task OPEN{d: [2024-09-27_201849_Fri-do, 2024-09-27_201931_Fri ], prio: 2024-11-01 12:00}
-3.14	test/priorities/2024-10-02_200923_Wed-some OPEN{ today , d:2024-09-29_231231_Sun-meta}
1.42	test/priorities/2024-09-27_201910_Fri-do bar OPEN{prio: 1, depends: 2024-09-27_201849}
2.38	test/priorities/reprocessing/2024-10-03_173610_Thu-README OPEN{prio: 2}
4.92	test/priorities/2024-10-02_193929_Wed-some OPEN{occasionally}
5.40	test/priorities/2024-09-27_212233_Fri-do OPEN{d: reprocessing}
5.42	test/priorities/erroneous/2024-09-30_111329_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931_Mon]}
5.42	test/priorities/erroneous/2024-09-30_111330_Mon-foo OPEN{d: [2024-09-27_201931_,2024-09-27_201931]}
5.42	test/priorities/erroneous/2024-09-30_111331_Mon-foo OPEN{d: [foobar]}
5.42	test/priorities/erroneous/2024-09-30_111332_Mon-duplicate 2 OPEN
5.42	test/priorities/erroneous/2024-09-30_111332_Mon-duplicate OPEN
5.42	test/priorities/erroneous/2024-09-30_111332_Mon-foo OPEN{d: [2024-09-27]}
9.48	test/priorities/foo/2024-10-24_153947_Thu-foo (for Wed OPEN{10-27})
